src/DecoratedStockChart.js

```diff
@@ -552,6 +555,24 @@
                          * remove attr from state
                          */
                         securityAttrPair[1].splice(securityAttrPair[1].indexOf(attr), 1);
+                        scope.checkForSecurityAttrMap();
+
+                    };
+
+
+                    /**
+                     * Hides the defaultSecurityAttribute input box when the multiple attributes added. It also changes the title
+                     * This is fired off when adding or removing attribute.
+                     */
+                    scope.checkForSecurityAttrMap =  function(){
```

Comment:
I see, so we want to automatically change the main Title when the user have select different securityAttributes && hide the defaultAttributee inputbox. 

From a coding perspective - this is somewhat unclean - as we are introducing a method that do two completely unrelated things (hide input box+ change chart title)

are you sure this is the best way? your `checkForSecurityAttrMap()` function has side-effects on the Highchart object as well as mutate a property on `scope` - you are duplicating state variables in multiple places. Could you re-think this a little? i.e. is `scope.multiTitle` really needed?

Finally, the name `checkForSecurityAttrMap` may not be the best possible name for this method because 1- it is unclear "what is being checked" and two the word check does not imply mutation of the directive's state - but the method mutates state.